rest-java/src/test/java/org/hiero/mirror/restjava/mapper/NetworkStakeMapperTest.java

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package org.hiero.mirror.restjava.mapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.hiero.mirror.common.domain.DomainBuilder;
+import org.hiero.mirror.common.domain.addressbook.NetworkStake;
+import org.hiero.mirror.rest.model.NetworkStakeResponse;
+import org.hiero.mirror.restjava.util.NetworkStakeUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class NetworkStakeMapperTest {
+
+    private DomainBuilder domainBuilder;
+    private NetworkStakeMapper mapper;
+
+    @BeforeEach
+    void setup() {
+        domainBuilder = new DomainBuilder();
+        mapper = new NetworkStakeMapperImpl();
+    }
+
+    @Test
+    void map() {
+        // given
+        final var stake = domainBuilder.networkStake().get();
+        stake.setNodeRewardFeeDenominator(0L);
+        stake.setNodeRewardFeeNumerator(100L);
+        stake.setStakingRewardFeeNumerator(10L);
+        stake.setStakingRewardFeeDenominator(100L);
```

Comment:
Use customize method and omit default values.
```suggestion
        final var stake = domainBuilder.networkStake().customize(n -> n.stakingRewardFeeNumerator(10L)).get();
```

Reply:
Addressed